### PR TITLE
feat:settings page implemented

### PR DESCRIPTION
**Pull request by albin1616**
setting page is implemented

### CODE REVIEW

**Review by manvithalva12 — CHANGES_REQUESTED** [review]
albin please add the software update section

File: settings.html [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Settings Page</title>
+    <style>
+        body {
+            font-family: Arial, sans-serif;
+        }
+        .container {
+            max-width: 600px;
```
Comment (Rajharsh2) [inline settings.html:12]: let width be auto

**Review by manvithalva12 — CHANGES_REQUESTED** [review]
albin please add background color

File: settings.html [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+<body>
+    <div class="container">
+        <h1>Settings</h1>
+        <form>
+            <label for="username">Username:</label>
+            <input type="text" id="username" name="username" placeholder="Enter your username">
+
+            <label for="language">Preferred Language:</label>
+            <select id="language" name="language">
+                <option value="en">English</option>
```
Comment (manvithalva12) [inline settings.html]: pls add malyalam

**Review by manvithalva12 — CHANGES_REQUESTED** [review]

File: settings.html (hunk 2 above)
Comment (albin1616) [reply settings.html]: resolved

**Review by manvithalva12 — APPROVED** [review]
resolved

**Pull request merged**